### PR TITLE
EDU-501 Make consent text editable on admin page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-501

### CODE REVIEW

File: migrations/educandu-2022-04-11-01-add-lessonsMode-to-rooms.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable camelcase, no-await-in-loop, no-console */
-export default class Educandu_2023_04_11_01_add_lessonsMode_to_rooms {
+export default class Educandu_2022_04_11_01_add_lessonsMode_to_rooms {
```
Comment (ahelmberger) [inline migrations/educandu-2022-04-11-01-add-lessonsMode-to-rooms.js:2]: :smile:

Comment (adrianaluchian) [reply migrations/educandu-2022-04-11-01-add-lessonsMode-to-rooms.js:2]: _If I could turn back time_ 🎤  🎶

File: src/components/consent-drawer.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
 import CookieAlertIcon from './icons/general/cookie-alert-icon.js';
+import React, { useEffect, useState, useRef, useMemo } from 'react';
 import { getCookie, setLongLastingCookie } from '../common/cookie.js';
 
-const CONSENT_COOKIE_VERSION = 'm157c4noq4et';
-
+const generateCookieHash = consentTextInAllLanguages => {
+  return md5(JSON.stringify(consentTextInAllLanguages));
+};
 export default function ConsentDrawer() {
```
Comment (ahelmberger) [inline src/components/consent-drawer.js:17]: Maybe we could have an empty line here, if you're up to it.

**Review by ahelmberger — APPROVED** [review]

File: src/components/consent-drawer.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/consent-drawer.js:17]: 🤦‍♀️  One of my plugins often makes me think empty lines are there where they aren't. Will add!

Screen Shot 2022-04-15 at 18 59 02

**Pull request merged**